README.md

```diff
@@ -417,6 +417,37 @@ $builder
     );
 ```
 
+### ScriptSort (Custom Script Sorting)
```

Comment:
Hi, sorry for the late review. Any chance you could add a link to the elasticsearch docs in the readme relevant for this sort?